.travis.yml

```diff
@@ -21,7 +21,6 @@ before_install:
 
 install:
 - npm install
-- npm install -g gulp
```

Comment:
Can we officially switch to yarn? 😀

Reply:
I've never used it before, what would it give us?

Reply:
Yarn is pretty nice, I'd recommend giving it a shot. Since it's newer, it benefits from an improved architecture in that it uses a lock file kind of like `npm shrinkwrap` except that it uses semver where possible to avoid baking the entire dependency tree in. It's also something like 2x-5x faster than `npm install`. And, anecdotally, it is just less buggy.